package.json

```diff
@@ -10,7 +10,7 @@
     "start": "nest start",
     "start:dev": "nest start --watch",
     "start:debug": "nest start --debug --watch",
-    "start:prod": "node dist/main",
+    "start:prod": "node dist/src/main",
```

Comment:
This is entirely dependent on your application. Usually this happens when you import a file from outside of `src` due to how typescript tries to keep the current file structure. In a `nest new` project, a `nest build && start:prod` works exactly as it should, so this change is not necessary

Reply:
I have this problem exactly when I use `nest new project`. At default project structure created with the command, `main.ts` is placed in `src` directory that make build in `dist/src/main.js`.

For default structure I think it's better has right script that points to default build directory.

Reply:
`nest new test-new-approach -p pnpm && cd test-new-approach && pnpm build && pnpm start:prod` works exactly as expected with no modifications. Run it inside of docker if you want to be certain.

Reply:
And just because `src/main.ts` is the default in a nest new project, the dist ends up as `dist/main.js` **unless** you end up importing from _outside_ of the `src`.**That's** when `dist/src` gets created

Reply:
> unless you end up importing from outside of the src.That's when dist/src gets created

Thanks, bro, I have config from outside of src, it makes create `dist/src/`.

I think I can close the Pull request.